### DISCUSSION TITLE
[Notify] No way to set icon color

### DISCUSSION THREAD

**Discussion by lyle45** [quasarframework/quasar#12417] [Ideas / Proposals] [answered]
[Notify] No way to set icon color in Notify plugin

Expected: 
Have a property for icon color in create options

**Comment by hawkeye64** [comment]
https://github.com/quasarframework/quasar/pull/12421

**Comment by rstoenescu** [comment, marked as answer]
New props will be available in Quasar v2.5.5: icon-color, icon-size, spinner-color, spinner-size.
